File: frontend/src/modules/et-boq-exchange/manifest.ts
import { Flag } from 'lucide-react';
import type { ModuleManifest } from '../_types';

export const manifest: ModuleManifest = {
  id: 'et-boq-exchange',
  name: 'Ethiopian BOQ Exchange',
  description: 'Import/export utilities for MoWUD standard BOQs',
  version: '1.0.0',
  icon: Flag,
  category: 'regional',
  defaultEnabled: true,
  depends: ['boq'],

  routes: [],
  navItems: [],
  translations: {
    en: {
      'et_pack.name': 'Ethiopia Pack',
      'et_pack.mowud_standard': 'MoWUD Standard',
    },
  },
};

File: backend/app/modules/ethiopia_pack/manifest.py
"""Module manifest for oe_ethiopia_pack."""

from app.core.module_loader import ModuleManifest

manifest = ModuleManifest(
    name="oe_ethiopia_pack",
    version="1.0.0",
    display_name="Regional Pack — Ethiopia",
    description=(
        "Ethiopian construction standards: MoWUD Standard Technical Specifications, "
        "ETB currency, metric units, and Ethiopian VAT rules."
    ),
    author="RandomCreatives / DigitalMehandis",
    category="regional",
    depends=[],
    auto_install=False,
    enabled=True,
)
